app/models/podcast_import.rb

```diff
@@ -262,9 +280,15 @@ def create_story(entry, series)
       explicit: explicit(entry[:itunes_explicit])
     )
 
-    # add the audio
-    if enclosure = enclosure_url(entry)
-      audio = version.audio_files.create!(label: 'Segment A', upload: enclosure)
+    if config[:audio] && config[:audio][entry.entry_id]
```

Comment:
that's right, this isn't requiring the config include entries for every episode.
e.g. you may only be replacing a single segment with another, clean/edited segment, and not have to do that for every one.

I don't think it throws errors, it would set a status and message on the version.